_layouts/home.html

```diff
@@ -7,8 +7,8 @@
         {% include navbar.html %}
         <div id="home-content" class="container">
 
-            {% include supportapache.html %}
             {% comment %}
+            {% include supportapache.html %}
```

Comment:
No particular issue with hiding this, just curious about the reasoning? Is it because we're now listing Apache events in the "community" menu?